test/cmdlineargs.jl

```diff
@@ -206,6 +206,7 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
     # We want to test oversubscription, but on manycore machines, this can
     # actually exhaust limited PID spaces
     cpu_threads = max(2*cpu_threads, min(50, 10*cpu_threads))
+    println("cpu_threads = ", cpu_threads, ", Sys.WORD_SIZE = ", Sys.WORD_SIZE)
```

Comment:
This is a known failure unrelated to this PR, so not neccessary to debug this here.

Reply:
@simeonschaub Thanks, could you point me to the place this is discussed?

Reply:
Should now be fixed by https://github.com/JuliaLang/julia/pull/39854